### PR TITLE
DOC-193 Clarified legacy vs current camera modes

### PR DESCRIPTION
**Pull request by jennaingersoll**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

File: libraries/shared/src/shared/Camera.cpp [hunk 1]
```diff
@@ -25,38 +25,45 @@
...
+ *       <td><p>The camera is positioned in front of your avatar. The camera moves and rotates independently from your avatar.
+ *       Your avatar's head is always facing the camera.</p><p><em>Default "look at myself" camera mode.</em></p></td>
  *     </tr>
  *     <tr>
  *       <td><strong>Mirror</strong></td>
  *       <td><code>"mirror"</code></td>
- *       <td>The camera is positioned such that you are looking directly at your avatar. The camera moves and rotates with your 
- *       avatar.</td>
+ *       <td><p>The camera is positioned such that you are looking directly at your avatar. The camera moves and rotates with
+ *       your avatar.</p>
```
Comment (shanzzam) [inline libraries/shared/src/shared/Camera.cpp]: Mirror mode did not allow for the camera to move or rotate. You were locked into that view.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22346/HighFidelity-Beta-PR16457-27d4bfd.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22346/HighFidelity-Quest-Beta-PR16457-27d4bfd.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22346/HighFidelity-Beta-PR16457-27d4bfd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22346/HighFidelity-Beta-Interface-PR16457-27d4bfd.dmg

File: libraries/shared/src/shared/Camera.cpp [hunk 2]
```diff
@@ -25,38 +25,45 @@
...
  *     </tr>
  *     <tr>
  *       <td><strong>Third&nbsp;Person</strong></td>
  *       <td><code>"third&nbsp;person"</code></td>
- *       <td>The camera is positioned such that you have a view from just behind your avatar. The camera moves and rotates with
- *       your avatar.</td>
+ *       <td><p>The camera is positioned such that you have a view from just behind your avatar. The camera moves and rotates 
+ *       with your avatar.</p>  
+ *       <p><em>Legacy third person camera camera mode.</em> To re-enable this behavior:</p> 
+ *       <pre class="prettyprint"><code>Camera.mode = "third person";</code></pre></td>
```
Comment (ctrlaltdavid) [inline libraries/shared/src/shared/Camera.cpp]: I don't think you need the "To re-enable this behaviour" explanation: each behavior is set by assigning the documented value to `Camera.mode`.

File: libraries/shared/src/shared/Camera.cpp [hunk 3]
```diff
@@ -25,38 +25,45 @@
...
  *     </tr>
  *     <tr>
  *       <td><strong>Mirror</strong></td>
  *       <td><code>"mirror"</code></td>
- *       <td>The camera is positioned such that you are looking directly at your avatar. The camera moves and rotates with your 
- *       avatar.</td>
+ *       <td><p>The camera is positioned such that you are looking directly at your avatar. The camera moves and rotates with
+ *       your avatar.</p> 
+ *       <p><em>Legacy "look at myself" behavior.</em> To re-enable this behavior: </p>
+ *       <pre class="prettyprint"><code>Camera.mode = "mirror";</code></pre></td>
```
Comment (ctrlaltdavid) [inline libraries/shared/src/shared/Camera.cpp]: Ditto re not needing "To re-enable" explanation.

File: libraries/shared/src/shared/Camera.cpp [hunk 4]
```diff
@@ -25,38 +25,45 @@
...
+ *       avatar's head always faces the camera look at point.</p>
+ *       <p><em>Default third person camera mode.</em></td>
  *     </tr>
  *     <tr>
  *       <td><strong>Selfie</strong></td>
  *       <td><code>"selfie"</code></td>
- *       <td>The camera is positioned in front of your avatar. The camera moves and rotates independently from your avatar.
- *       Your avatar's head is always facing the camera.</td>
+ *       <td><p>The camera is positioned in front of your avatar. The camera moves and rotates independently from your avatar.
+ *       Your avatar's head is always facing the camera.</p><p><em>Default "look at myself" camera mode.</em></p></td>
```
Comment (ctrlaltdavid) [inline libraries/shared/src/shared/Camera.cpp]: Would be clearer to start the new `<p>` on a new line.

File: libraries/shared/src/shared/Camera.cpp (hunk 1 above)
Comment (jennaingersoll) [reply libraries/shared/src/shared/Camera.cpp]: Fixed!

File: libraries/shared/src/shared/Camera.cpp (hunk 4 above)
Comment (jennaingersoll) [reply libraries/shared/src/shared/Camera.cpp]: Done.

File: libraries/shared/src/shared/Camera.cpp (hunk 3 above)
Comment (jennaingersoll) [reply libraries/shared/src/shared/Camera.cpp]: Removed.

File: libraries/shared/src/shared/Camera.cpp (hunk 2 above)
Comment (jennaingersoll) [reply libraries/shared/src/shared/Camera.cpp]: Removed.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22350/HighFidelity-Beta-PR16457-499a0a8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22350/HighFidelity-Beta-Interface-PR16457-499a0a8.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22350/HighFidelity-Beta-PR16457-499a0a8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22350/HighFidelity-Quest-Beta-PR16457-499a0a8.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22350/HighFidelity-Beta-PR16457-499a0a8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22350/HighFidelity-Beta-Interface-PR16457-499a0a8.dmg

File: libraries/shared/src/shared/Camera.cpp (hunk 2 above)
Comment (ctrlaltdavid) [reply libraries/shared/src/shared/Camera.cpp]: Should also remove the `<pre>` paragraph, here and below.

**Pull request merged**